<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesWidget</class>
 <widget class="QWidget" name="PreferencesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>311</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QFrame" name="boxPreferencesFont">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="lblPrefWindowsFont">
          <property name="text">
           <string>Windows font</string>
          </property>
          <property name="buddy">
           <cstring>cmbPrefWindowsFont</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QFontComboBox" name="cmbPrefWindowsFont"/>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinPrefWindowsFontSize">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>28</number>
          </property>
          <property name="value">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblPrefTerminalFont">
          <property name="text">
           <string>Terminal window font</string>
          </property>
          <property name="buddy">
           <cstring>cmbPrefTerminalFont</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QFontComboBox" name="cmbPrefTerminalFont">
          <property name="statusTip">
           <string>Font for Telnet client utility </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lblPrefFontSize">
          <property name="text">
           <string>Font size</string>
          </property>
          <property name="buddy">
           <cstring>spinPrefWindowsFontSize</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spinPrefTerminalFontSize">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>28</number>
          </property>
          <property name="value">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="boxPreferencesIcon">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkPrefMinToTray">
          <property name="text">
           <string>Icon in the system tray</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkPrefAnimateIcon">
          <property name="toolTip">
           <string>Animate tray icon when working</string>
          </property>
          <property name="statusTip">
           <string>Animate tray icon when working</string>
          </property>
          <property name="text">
           <string>Animate tray icon when working</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkPrefChangeIconDie">
          <property name="text">
           <string>Change icon when a host dies</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="lineWidth">
           <number>2</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkPrefAlwaysTop">
          <property name="text">
           <string>HostMonitor always Top</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="boxPreferencesAutosave">
       <property name="title">
        <string>Autosave option</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>16</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <item>
         <widget class="QComboBox" name="cmbPrefAutoSaveOptions">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Do not save TestList automatically</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Save tests and stats at midnight</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Auto save TestList after any changes</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkPrefAutoSaveOptions">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Auto save windows options</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chkPrefRecent">
            <property name="text">
             <string>Keep a list of </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinPrefRecent">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblPrefRecent">
            <property name="text">
             <string>recent addresses</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>7</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="boxPreferencesAlert">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chkPrefStoreStatHML">
            <property name="text">
             <string>Store statistics in HML file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkPrefStoreRecurencesHML">
            <property name="text">
             <string>Store &quot;Recurrences&quot; in HML file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkPrefRememberCurrentFolder">
            <property name="text">
             <string>Remember current folder</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
